code/rdb/datastripe.q

```diff
@@ -25,8 +25,6 @@ initdatastripe:{
     t:tables[`.] except .rdb.ignorelist;
     .rdb.access:([table:t] start:.ds.getstarttime each t; end:0Np ; stptime:0Np ; keycol:`sym^.rdb.tablekeycols[t]);
     modaccess[.rdb.access];
-
     };
 
 if[.ds.datastripe;.proc.addinitlist[(`initdatastripe;`)]];
-
```

Comment:
restore the blank lines removed from this file (just to keep the PR diff as clean as possible)